### ISSUE TITLE
This line of code has possibility to cause a `RangeError: Maximum call stack size exceeded at String.match ()`

### ISSUE THREAD

**Issue by Paper-Folding** [greensock/GSAP#545]
Here is the final location that caused the error:  
https://github.com/greensock/GSAP/blob/472d4170d4aa3cceb8a3880d7cad700f22fbc545/src/CSSPlugin.js#L322

I met a `RangeError: Maximum call stack size exceeded` on my production site, this issue does not happen very often, but it happened indeed.  

I must apologize that I can't provide a very detailed issue report now (because I can't locate where it is actual called in my code under a production mode), now I unminimized my code and add source map, once this issue happens again, I'll report more details here.

**Comment by Paper-Folding** [comment]
Okay, the call stack(I analyzed with https://github.com/sokra/source-map-visualization):  
https://github.com/greensock/GSAP/blob/472d4170d4aa3cceb8a3880d7cad700f22fbc545/src/CSSPlugin.js#L322
https://github.com/greensock/GSAP/blob/472d4170d4aa3cceb8a3880d7cad700f22fbc545/src/CSSPlugin.js#L1095
https://github.com/greensock/GSAP/blob/472d4170d4aa3cceb8a3880d7cad700f22fbc545/src/gsap-core.js#L2204

**Comment by Paper-Folding** [comment]
And it's so weird that the issue happens in `CSSPlugin.js`, I did not even explicitly use `CSSPlugin.js` in my source file, I use: 
```js
```

**Comment by jackdoyle** [comment]
Hm, that sounds odd. Can you please provide any (or all) of the following?: 

1. A minimal demo illustrating the issue
2. The exact error message. Was it only "RangeError: Maximum call stack size exceeded at String.match()"? Any other clues? 
3. The value(s) you're feeding into the tween that's triggering that error. I wonder if it's a MASSIVE string with hundreds of thousands of numbers or something.

I just can't imagine why that line would throw the error. Nobody else has reported anything similar. I'd love to see a demo, though.

**Comment by jackdoyle** [comment]
And to answer your question about CSSPlugin, that's automatically included in the core because almost everybody ends up wanting to tween CSS-related values. If you truly have no such needs at all in your app, yes, you technically should be able to import gsap from "gsap/gsap-core" which would exclude CSSPlugin (but again, that's almost never a good idea). The fact that you're getting the error in that CSSPlugin file tells me that you actually are trying to animate something on a DOM element, likely CSS-related, so you shouldn't be excluding CSSPlugin.

**Comment by Paper-Folding** [comment]
This issue is just too rare to happen, and only happened once on production mode, the full stack trace is(very hard too identify, I think you don't actually wanna see this, this is from a packed js by vite.js):
```
RangeError: Maximum call stack size exceeded
at String.match ()
at Ao.r.Il (http://192.168.0.2/assets/light-box-6326ac03.js:3:76892)
at Ao.r.init (http://192.168.0.2/assets/light-box-6326ac03.js:3:87873)
at h (http://192.168.0.2/assets/light-box-6326ac03.js:3:55433)
at wo (http://192.168.0.2/assets/light-box-6326ac03.js:3:25511)
at za (http://192.168.0.2/assets/light-box-6326ac03.js:3:26174)
at e.render (http://192.168.0.2/assets/light-box-6326ac03.js:3:58961)
at new t (http://192.168.0.2/assets/light-box-6326ac03.js:3:58771)
at t.set (http://192.168.0.2/assets/light-box-6326ac03.js:3:62304)
at Vr (http://192.168.0.2/assets/light-box-6326ac03.js:3:99994)
```
So I analyze the stack trace utilizing a source file map visualization map, but it is too time comsuming to find all stack trace using this method, so I unminized source code and waiting for it to happen again possibility.
By the way, the error is happened on Opera for Android arm64, version 75.4.3978.72990.

I don't want to waste your developers' time here, so you may just ignore this issue for now, once it happens again, I'll report it immediately. 

And appreciate for your great work, thank you!

**Comment by Paper-Folding** [comment]
I finally resolved this error, this error happens because my Draggable.js is using a low version cdn while gsap is the newest high version in my production site (using cdn, so I forget to update url). In development mode, I'm using npm so the version is always the latest, so that's way I can't catch a error in development mode.

**Comment by jackdoyle** [comment]
Huh, very interesting. I still don't understand why having an old version of Draggable would cause that line to error in CSSPlugin. Are you sure things are resolved? I mean you should definitely be using a more recent version...I just want to make sure the real issue was resolved.

**Comment by Paper-Folding** [comment]
> Huh, very interesting. I still don't understand why having an old version of Draggable would cause that line to error in CSSPlugin. Are you sure things are resolved? I mean you should definitely be using a more recent version...I just want to make sure the real issue was resolved.

Yes, long story short: 3.11 Draggable.js incompatible with 3.12 Gsap.js, when draggable.kill() method is called, this error happened.

**Comment by Paper-Folding** [comment]
Oh, wait, it is Inertia Plugin that caused this error, finally I got a re-production demo: https://codepen.io/paper-folding/pen/MWGBVer
Just drag the block and wait, the issue will happen.

**Comment by jackdoyle** [comment]
For the record, there was a regression in 3.12.0 that could cause an error like this (not from that line you mentioned though) if you create a Draggable, enable inertia, and then call disable() on that Draggable instance. It should be resolved in the next release, 3.12.2 due out soon. I just wonder if maybe that was the root cause here.

**Comment by jackdoyle** [comment]
@Paper-Folding can you share any details about what exactly resolved it on your end? I don't think this had anything to do with GSAP specifically, but I'm curious about what solved the problem for you.

**Comment by Paper-Folding** [comment]
> @Paper-Folding can you share any details about what exactly resolved it on your end? I don't think this had anything to do with GSAP specifically, but I'm curious about what solved the problem for you.

Well, if you insist.... Intertia plugin 3.11 does not compatible with Draggable 3.12 (especially when `.destroy` method is called on Draggable instance). That's all I have tested. And I really appreciate your work, I must apologize for being so implicit above.

**Comment by Paper-Folding** [comment]
And I resolved the problem by removing Inertia Plugin, users weren't actually allergic to drag inertia a lot, they only cares drag.

**Comment by jackdoyle** [comment]
Yes, that issue was resolved in 3.12.2 of Draggable as I mentioned above. If you become aware of any other issues, please do let us know.
